core/rend/TexCache.cpp

```diff
@@ -395,28 +395,28 @@ void BaseTextureCacheData::ComputeHash()
 		// We use the old size to compute the hash for backward-compatibility
 		// with existing custom texture packs.
 		int oldSize = width * height / 8;
-		old_vqtexture_hash = XXH32(&vram[mmStartAddress - VQ_CODEBOOK_SIZE], oldSize, 7);
+		old_vqtexture_hash = XXH3_64bits(&vram[mmStartAddress - VQ_CODEBOOK_SIZE], oldSize);
```

Comment:
Changing the hash value will invalidate all existing texture packs. Obviously we don't want that.